com.unity.netcode.gameobjects/Runtime/com.unity.netcode.runtime.asmdef

```diff
@@ -24,6 +24,11 @@
             "name": "com.unity.multiplayer.tools",
             "expression": "",
             "define": "MULTIPLAYER_TOOLS"
+        },
+        {
+            "name": "com.unity.multiplayer.tools",
+            "expression": "1.0.0-pre.4",
```

Comment:
Can you do >= instead of = (which is the implicit operator with no operator is provided). That way this code doesn't break if we release a pre.5

Reply:
The default is actually >= in the version define, see screenshot: 
![image](https://user-images.githubusercontent.com/682952/152865443-3e8dd0a0-a7a3-4f9b-a61d-2d1f57eb3dad.png)

Reply:
Ah nice, my bad!